app/src/main/java/com/jerboa/ui/components/post/create/CreatePost.kt

```diff
@@ -74,11 +75,18 @@ fun CreatePostHeader(
                         color = MaterialTheme.colorScheme.onSurface,
                     )
                 } else {
-                    // Todo add are you sure cancel dialog
-                    Icon(
-                        Icons.Outlined.Add,
-                        contentDescription = stringResource(R.string.form_submit),
-                    )
+                    // Todo add are you sure cancel dialog+
+                    if (formValid && !loading) {
+                        Icon(
+                            Icons.Filled.Send,
```

Comment:
Would be much easier to do `icon = if (formValid) { ...`

Reply:
Yep, that's much cleaner